src/plugins/intel_gna/tests/functional/fp32_inference_tests/test_models.cpp

```diff
@@ -0,0 +1,398 @@
+    auto split = std::make_shared<opset10::Split>(scaleshift_add->output(0), split_axis->output(0), 2);
+
+    // concat
+    auto concat = std::make_shared<opset10::Concat>(OutputVector{split->output(0), split->output(1)}, 1);
+
+    // fully connected
+    auto fc_weights = std::make_shared<opset10::Constant>(precision, Shape{20, 10}, std::vector<float>(200, 1.0f));
+    auto fc_biases = std::make_shared<opset10::Constant>(precision, Shape{1, 10}, std::vector<float>(10, 1.0f));
+    auto fc_before_biases = std::make_shared<opset10::MatMul>(concat->output(0), fc_weights->output(0));
+    auto fc = std::make_shared<opset10::Add>(fc_before_biases->output(0), fc_biases->output(0));
+
+    auto function = std::make_shared<Model>(fc, ParameterVector{input}, "InputSplitConcatModel");
+    return function;
+}
+
+std::shared_ptr<ov::Model> power_with_scale_factor_model() {
+    auto precision = element::Type_t::f32;
+
+    // input
+    auto input = std::make_shared<opset10::Parameter>(precision, Shape{1, 10});
+
+    // power
+    auto exponents = std::make_shared<opset10::Constant>(precision, Shape{1, 10}, std::vector<float>(10, 1.0f));
+    auto power = std::make_shared<opset10::Power>(input->output(0), exponents->output(0));
+
+    // fully connected
+    auto fc_weights = std::make_shared<opset10::Constant>(precision, Shape{10, 12}, std::vector<float>(120, 1.0f));
+    auto fc_biases = std::make_shared<opset10::Constant>(precision, Shape{1, 12}, std::vector<float>(12, 1.0f));
+    auto fc_before_biases = std::make_shared<opset10::MatMul>(power->output(0), fc_weights->output(0));
+    auto fc = std::make_shared<opset10::Add>(fc_before_biases->output(0), fc_biases->output(0));
+
+    auto function = std::make_shared<Model>(fc, ParameterVector{input}, "PowerWithScaleFactor1");
+    return function;
+}
+
+std::shared_ptr<Model> lstm_cell_only_model() {
+    auto precision = element::Type_t::f32;
+
+    // input
+    auto input = std::make_shared<opset10::Parameter>(precision, Shape{1, 96});
+
+    // scaleshift
+    auto scaleshift_mul_constant =
+        std::make_shared<opset10::Constant>(precision, Shape{1, 96}, std::vector<float>(96, 0.10f));
+    auto scaleshift_mul = std::make_shared<opset10::Multiply>(input->output(0), scaleshift_mul_constant->output(0));
+    auto scaleshift_add_constant =
+        std::make_shared<opset10::Constant>(precision, Shape{1, 96}, std::vector<float>(96, 0.10f));
+    auto scaleshift_add = std::make_shared<opset10::Add>(scaleshift_mul->output(0), scaleshift_add_constant->output(0));
+
+    // split
+    auto split_axis = std::make_shared<opset10::Constant>(element::Type_t::i32, Shape{}, std::vector<int32_t>{1});
+    auto split = std::make_shared<opset10::Split>(scaleshift_add->output(0), split_axis->output(0), 3);
+
+    // LSTM cell
+    auto lstm_blob_w = std::make_shared<opset10::Constant>(precision, Shape({128, 32}), std::vector<float>(4096, 0.1f));
```

Comment:
GFYI, If the version of opset is not important in the test it would be better to use: using namespace ov::ops::opset10
and std::make_shared<Split>